### ISSUE TITLE
Question on Microsoft.Data.Analysis package development

### ISSUE THREAD

**Issue by chriss2401** [dotnet/machinelearning#5870]
I just have a simple question about this package. It recently seems to be have moved here but development is slow and there hasn't been a new version of it published for ~14 months. Is it on EOL (end of life) support or is there development planned for the future ?

**Comment by michaelgsharp** [comment]
Its not EOL and we have just released a new version last week.

@briacht @pgovind any comments?

**Comment by briacht** [comment]
Hey @chriss2401, we do plan on continuing to work on DataFrames, and this will especially pick up as we develop our plan for data prep / wrangling / exploration for ML.NET.

Are there are particular features that are currently missing that you'd like to see in the future?

**Comment by niko86** [comment]
I'm not the original poster but i would like to see more string based methods in the future. Currently I'm scratching my head trying to work out how to filter a string column that contains a certain substring.

**Comment by tombohub** [comment]
@niko86 how do you find this DataFrame compared to python pandas?

**Comment by niko86** [comment]
> @niko86 how do you find this DataFrame compared to python pandas?

Obviously, it's much earlier in development than pandas so it's unfair for me to judge it too harshly. However, a lot of work needs to go into this before I could go from using pandas in python to using this package in c#.

**Comment by rosebyte** [comment]
Closing as the question was answered.
